### PR TITLE
Added ACL 2020 awards

### PR DESCRIPTION
**Pull request by mjpost**
This picks up from an old conversion (#240). I added the ACL 2020 awards, and made a first attempt at the display. The awards display looks like this in the list display:

image

and this on the paper page:

image

### CODE REVIEW

**Comment by akoehn** [comment]
Nifty implementation, I like it. I am torn about having the award icon in the text as well -- it is the only image in that table.

**Comment by mjpost** [comment]
Ugh, sorry for the repeated pushes. It is tricky to ensure that experimental changes I make in the `build/` directory get propagated back into the git repo, once working.

(@akoehn I wonder about populating `build/` with links to the hugo layout files, instead of copying? Then any changes in the test environment would automatically be seen by git).

**Comment by mjpost** [comment]
Same here on the icon. I agree it sticks out, but on the other hand, the bit of flair does seem to be earned.

**Comment by akoehn** [comment]
The problem with creating links instead of copying is that publishing the site via rsync gets more complicated.  It will happily copy symlinks and then the target of the link will not be there on the target machine. Doing some fancy post processing before publishing (replace symlinks by real copies of the files) creates new possibilities to fail.

Maybe we can have a new target that creates symlinks for development purposes -- they would be automatically gone after re-building the site.

**Comment by mjpost** [comment]
`rsync -L` will copy symlinks as regular files. Wouldn't that be sufficient?

**Comment by akoehn** [comment]
Right, I've added this to my to-do list for tomorrow.

**Comment by akoehn** [comment]
I had a look at the symlink options and things are a bit more complicated.  Let's use #815 to discuss this further.

**Comment by mjpost** [comment]
I like this—it's still visible, but doesn't jump out to the eye as much. I also like the "award" icon much better than the trophy I had earlier.

image

image

**Comment by mjpost** [comment]
@pkolachi, you started entering awards, right? Do you have any interest in adding to this PR? (or if you have some data in a spreadsheet already, you could send it on)

**Comment by mjpost** [comment]
So...good to merge?

**Comment by akoehn** [comment]
We can merge this as-is, but the schema allows for an arbitrary number of awards while the code only shows the first one.  Maybe we should fix this now instead of being surprised in a decade or so.

I don't know how to write a `with` block in hugo templates for more than one element; we could simply have the schema.rnc define `award?` instead of `award*` so that the current limitation is obvious upon validating data.

**Comment by pkolachi** [comment]
> @pkolachi, you started entering awards, right? Do you have any interest in adding to this PR? (or if you have some data in a spreadsheet already, you could send it on)

Hej Matt,
The papers I found/added to the spreadsheet can be found here:
https://docs.google.com/spreadsheets/d/174uNP-wpyjU0Hx4CsT_tQPiDmU5b-5jHzCaEJMYfXZA/edit?usp=sharing

The first sheet is the best papers, and the second sheet is for honorable mentions and runner-ups? etc. I also included the source from where I compiled the information in the third sheet.  I must say I haven't done anything on this since last summer, so the recent conferences are not included :-)

**Comment by mbollmann** [comment]
> We can merge this as-is, but the schema allows for an arbitrary number of awards while the code only shows the first one. Maybe we should fix this now instead of being surprised in a decade or so.

Good catch! For multiple `<award>` fields to work, `award` should be added [to the `LIST_ELEMENTS` variable](https://github.com/acl-org/acl-anthology/blob/78010236ead960d1e36b501eff3b2b39784cc1a4/bin/anthology/data.py#L36), otherwise subsequent `<award>` tags would just overwrite the previous one. If there are allowed to be multiple awards this way, the Hugo template would need to be changed to use a `range` loop instead of a `with` block.

**Comment by mjpost** [comment]
All right, I fixed that.

@pkolachi, thanks for sharing the spreadsheet. I'm not going to backfill this myself; perhaps we can crowdsource them, or leave them on a case-by-case basis.

File: data/xml/N19.xml [hunk 1]
```diff
@@ -4800,6 +4800,7 @@
       <abstract>This paper seeks to model human language by the mathematical framework of quantum physics. With the well-designed mathematical formulations in quantum physics, this framework unifies different linguistic units in a single complex-valued vector space, e.g. words as particles in quantum states and sentences as mixed systems. A complex-valued network is built to implement this framework for semantic matching. With well-constrained complex-valued components, the network admits interpretations to explicit physical meanings. The proposed complex-valued network for matching (CNM) achieves comparable performances to strong CNN and RNN baselines on two benchmarking question answering (QA) datasets.</abstract>
       <url hash="3095278b">N19-1420</url>
       <doi>10.18653/v1/N19-1420</doi>
+			<award>Best Explainable NLP Paper</award>
```
Comment (mbollmann) [inline data/xml/N19.xml:4803]: Tab indenting?
(I'm almost surprised we don't yet have an automated check for this ;))

**Review by mbollmann — APPROVED** [review]
LGTM now!

File: data/xml/N19.xml (hunk 1 above)
Comment (akoehn) [reply data/xml/N19.xml:4803]: Just wait for my next PR :-)

Comment (mjpost) [reply data/xml/N19.xml]: Oh, man! That was an accident. I just fixed it. In doing so I turned up a number of abstracts with the same problem. We should fix `anthology.utils.indent` to remove them in that same PR.

**Comment by mjpost** [comment]
I am probably going to work through @pkolachi's spreadsheet and just add them to this PR (on the assumption that your checks didn't include actually double-checking that the awards were correct), so please let me merge.

**Comment by pkolachi** [comment]
I compared the information from proceedings front matter with conference website when available, so for papers listed in the sheet, I can say I've double checked. Though for conferences pre-2015, there were no Outstanding awards I think? and the ones listed in the sheet are the runners-up for the Best paper awards (since they are mentioned in the proceedings). I'm happy to help with back-addding entries this weekend ... if you think that's useful.

**Comment by akoehn** [comment]
I think we can merge these and create a tracking issue for missing old awards.

**Comment by mjpost** [comment]
Agreed, will do

matt (from mobile)

**Pull request merged**